pkg/api/v1alpha1/fluxconfig.go

```diff
@@ -4,10 +4,15 @@ import (
 	"errors"
 	"fmt"
 	"net/url"
+
+	"github.com/aws/eks-anywhere/pkg/logger"
 )
 
 const (
```

Comment:
```suggestion

const (
	Rsa   SshKeyAlgorithmType = "rsa"
	Ecdsa  SshKeyAlgorithmType = "ecdsa"
	Ed25519 SshKeyAlgorithmType = "ed25519"
)

type SshKeyAlgorithmType string
```

Reply:
will follow up on this for sure!

Reply:
https://github.com/aws/eks-anywhere/issues/2036